functions/daemon.go

```diff
@@ -113,14 +112,14 @@ func startGoRoutines(wg *sync.WaitGroup) context.CancelFunc {
 	if _, err := config.ReadNetclientConfig(); err != nil {
 		logger.Log(0, "error reading neclient config file", err.Error())
 	}
+	config.UpdateNetclient(*config.Netclient())
```

Comment:
what is the purpose of this line ....  ReadNetclientCconfig just updated the in-memory copy of the configuration.

Reply:
in last meeting, I mentioned this does not work. maybe you can fix it, but the config is incorrect after editing and restarting daemon